front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/application/Application.groovy

```diff
@@ -32,31 +32,40 @@ import groovy.transform.stc.SimpleType
 import groovy.util.logging.Slf4j
 import org.springframework.validation.Errors
 
-import java.lang.reflect.Field
-import java.lang.reflect.Modifier
-
 @ToString
 @Slf4j
 class Application {
   String name
   String description
   String email
-  String owner
-  String type
-  String group
-  String monitorBucketType
-  String pdApiKey
+  String accounts
   String updateTs
   String createTs
-  String tags
-  String regions
-  String accounts
-  String repoProjectKey
-  String repoSlug
-  String repoType
-  String cloudProviders
-  String platformHealthOnly
-  String platformHealthOnlyShowOverride
+
+  private Map<String, Object> details = new HashMap<String, Object>()
+
+  @JsonAnyGetter
+  public Map<String,Object> any() {
```

Comment:
Minor, but any() seems like a strange method name ... is `details()` a bad choice?